mantidproject/mantid
# Fix images for patch plot images on ConvertToReflectometryQ docs

Fixes #13668

To test:
- go to [ConvertToReflectometryQ Doc page](http://docs.mantidproject.org/nightly/algorithms/ConvertToReflectometryQ-v1.html)
- check that patch plots for K and P transformations have images.
